scripts/core/editor3/actions/editor3.jsx

```diff
@@ -0,0 +1,21 @@
+export function onChangeEditorState(editorState) {
```

Comment:
There is some stuttering here. We are in the editor component and this is already known. On top of that, `onChange` is common draft-js implementation approach so it should be enough to call this `onChange` instead of `onChangeEditorState`.